README.md

```diff
@@ -19,36 +19,27 @@ My team uses a git commit message convention that helps us read fast and also is
 
 We agree on a short list of leading active verbs:
 
-* Add = Create a capability e.g. feature, test, dependency.
-
-* Cut = Remove a capability e.g. feature, test, dependency.
-
-* Fix = Fix an issue e.g. bug, typo, accident, misstatement.
-
-* Bump = Increase the version of something e.g. dependency.
-
-* Make = Change the build process, or tooling, or infra.
-
-* Start = Begin doing something; e.g. create a feature flag.
-
-* Stop = End doing something; e.g. remove a feature flag.
-
-* Refactor = A code change that MUST be just a refactoring.
-
-* Reformat = Refactor of formatting, e.g. omit whitespace.
-
-* Optimize = Refactor of performance, e.g. speed up code.
-
-* Document = Refactor of documentation, e.g. help files.
-
+| Verb          | Description                                         |
```

Comment:
We aim for CommonMark and cmark parseable; these don't have Markdown tables. We've tried using HTML tables in many of our markdown files, and these work, however GitHub doesn't allow formatting. So generally we try to use lists instead of tables except when necessary.

Reply:
For reference: http://spec.commonmark.org/ --> 0.27 of 2016-11-18. GitHub flavored markdown has it: https://github.github.com/gfm/#tables-extension-